dhctl/pkg/kubernetes/actions/deckhouse/log.go

```diff
@@ -33,10 +33,12 @@ import (
 )
 
 var (
-	ErrListPods      = errors.New("No Deckhouse pod found.")
```

Comment:
We use upper letter because these errors output to log "as is" and they will look ugly for client.

Please revert it